scvelo/tools/terminal_states.py

```diff
@@ -227,10 +227,14 @@ def terminal_states(
     .. code:: python
 
         scv.tl.terminal_states(adata)
-        scv.pl.scatter(adata, color=[ 'root_cells', 'end_points'])
+        scv.pl.scatter(adata, color=['root_cells', 'end_points'])
 
     .. image:: https://user-images.githubusercontent.com/31883718/69496183-bcfdf300-0ecf-11ea-9aae-685300a0b1ba.png
 
+    Other option would be to use :func:`cellrank.tl.terminal_states` (end points)
+    or :func:`cellrank.tl.initial_states` (root points), which uses the real Schur decomposition
+    instead of the eigendecomposition because of the asymmetricity of the transition matrix.
+
```

Comment:
Better: "An alternative option would be to use XXX or XXX, both of which make use of real Schur decompositions to handle non-symmetries in the transition matrix. These methods are also able to separate multiple initial/terminal states and to pick the most likely cells from each. "